src/services/posts.rs

```diff
@@ -31,51 +34,96 @@ pub async fn restaurant(
     HttpResponse::Ok().json(Restaurant::get_all_resturants(&conn))
 }
 
+/*
 #[get("/restaurant/{id}")]
 pub async fn restaurant_by_id(
     pool: web::Data<Pool<ConnectionManager<SqliteConnection>>>,
     web::Path(id): web::Path<i32>,
 ) -> impl Responder {
     let conn = pool.get().unwrap();
 
-    HttpResponse::Ok().json(Restaurant::get_restaurant_by_id(id, &conn))
-}
-
-#[derive(Deserialize)]
-pub struct LatLngQuery {
-    northeast: String,
-    southwest: String,
-}
+}*/
 
 #[derive(Deserialize)]
-pub struct NameQuery {
-    name: String,
-}
-
-#[get("/restaurants/search")]
-pub async fn restaurants_search(
-    pool: web::Data<Pool<ConnectionManager<SqliteConnection>>>,
-    query: web::Query<LatLngQuery>,
-) -> impl Responder {
-    let conn = pool.get().unwrap();
-    let mut northeast = query.northeast.split(",");
-    let mut southwest = query.southwest.split(",");
-    let nelat = northeast.next().unwrap().parse::<f32>().unwrap();
-    let nelng = northeast.next().unwrap().parse::<f32>().unwrap();
-    let swlat = southwest.next().unwrap().parse::<f32>().unwrap();
-    let swlng = southwest.next().unwrap().parse::<f32>().unwrap();
-    HttpResponse::Ok().json(Restaurant::search_by_lat_lng(
-        nelat, nelng, swlat, swlng, &conn,
-    ))
+pub struct Restaurantsearchinput {
+    id: Option<i32>,
+    name: Option<String>,
+    city: Option<String>,
+    zip: Option<String>,
+    location: Option<String>, /*xx.yy,xx.yy,xx.yy,xx.yy*/
```

Comment:
What is the xx.yy thing?

Reply:
lat lng floats, så en location query for aalborg vil se sådan her ud
restaurant/search?location=57.064817,9.887307,57.014758,9.961513
de 2 første er lat lng for northwest og de 2 sidste er lat lng for southeast